### PR TITLE
Release/2.2.0

### PR DESCRIPTION
**Pull request by atis06**

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -1,4 +1,15 @@
 # Release History
+*****************
+## Release ONDEWO VTSI Client Python 2.2.0
+
+### Improvements
+
```
Comment (gaborod16) [inline RELEASE.md:9]: what are the improvements?
What was added in this new version?

File: examples/caller/make_multiple_calls.py [hunk 2]
```diff
@@ -1,13 +1,10 @@
...
-from threading import Thread
-from typing import List
 
 from ondewo.vtsi.client import VtsiClient
-from ondewo.vtsi.voip_pb2 import StartCallInstanceResponse
+from ondewo.vtsi.voip_pb2 import StartCallInstanceResponse, StartMultipleCallInstancesResponse
 
 #     SIP
-SIP_SIM_VERSION: str = "1.5.4"
+SIP_SIM_VERSION: str = "latest"
```
Comment (gaborod16) [inline examples/caller/make_multiple_calls.py]: don't we know the version for this?

File: setup.py [hunk 3]
```diff
@@ -8,7 +8,7 @@
 
 setuptools.setup(
     name="ondewo-vtsi-client",
-    version="2.1.0",
+    version="2.1.1",
```
Comment (gaborod16) [inline setup.py]: 2.2.0

File: Makefile [hunk 4]
```diff
@@ -41,4 +41,15 @@ generate_specific_protos:
...
+	echo 'pushed to pypi : )'
+
+build_package:
+	python setup.py sdist bdist_wheel
+
+upload_package:
+	twine upload -r pypi dist/*
+
+clear_package_data:
+	rm -rf build dist ondewo_logging.egg-info
```
Comment (gaborod16) [inline Makefile]: ondewo_logging?

**Review by gaborod16 — CHANGES_REQUESTED** [review]
Small changes, looks good overall.

File: Makefile (hunk 4 above)
Comment (atis06) [reply Makefile]: Ferg, idk

**Pull request merged**